test/models/post_test.rb

```diff
@@ -1,7 +1,34 @@
 require 'test_helper'
 
 class PostTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @user = User.create(name: 'John', posts_counter: 0)
+    @post = Post.create(user: @user, title: 'Test Post', comments_counter: 0, likes_counter: 0)
+  end
+
+  test 'post is valid' do
+    assert @post.valid?
+  end
+
+  test 'post must have a title' do
+    @post.title = nil
+    assert_not @post.valid?, 'Post should be invalid without a title'
+  end
+
+  test 'five most recent comments returns the correct number of comments' do
+    6.times { |n| @post.comments.create(content: "Comment #{n}") }
+
+    recent_comments = @post.five_most_recent_comments
+
+    assert_equal 5, recent_comments.length
+    assert_equal 'Comment 5', recent_comments.first.content
+  end
+
+  test 'updates the user posts counter' do
+    assert_equal 0, @user.posts_counter
+
+    @post.update_user_post_counter
+
+    assert_equal 1, @user.reload.posts_counter
```

Comment:
- Based on the project requirement, you're meant to add validation tests for the models based on the validations you added. For example, the validation test for "Title must not exceed 250 characters." is missin here. Kindly make sure all validation rules are tested and in a "describe" test case.

### Requirements:
![image](https://github.com/CarlosZ96/Blog-App-MCV/assets/50990925/46b79a7f-8f0d-4194-9f3b-f53d7f8c01ed)

### Sample of expected code:
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

- Kindly do these same changes to "User" model as well 👍